core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1990,20 +1994,16 @@ private void executeSubworkflowTaskAndSyncData(
         WorkflowSystemTask subWorkflowSystemTask =
                 systemTaskRegistry.get(TaskType.TASK_TYPE_SUB_WORKFLOW);
         subWorkflowSystemTask.execute(subWorkflow, subWorkflowTask, this);
-        // Keep Subworkflow task's data consistent with Subworkflow's.
```

Comment:
For my edification here: what are the ramifications of removing this functionality?

Reply:
None. The code is moot. It gets a reference to a map and does a `putAll` of the contents into the same map.